test/path-types.cooltt

```diff
@@ -4,13 +4,13 @@ def formation : {
   (A : 𝕀 → type) (a : A 0) (b : A 1) → type
 } = {
   A a b =>
-  path A a b
+  pathd A a b
```

Comment:
all the `pathd` in this file can be done by `path`, it seems...

Reply:
That's true. I intentionally didn't make that change so that this file was a more modest version of the tests in the other file (which tripped bugs that this one didn't). But I don't care if someone wishes otherwise.